common/src/main/java/org/apache/rocketmq/common/message/MessageConst.java

```diff
@@ -52,6 +52,10 @@
     public static final String PROPERTY_PUSH_REPLY_TIME = "PUSH_REPLY_TIME";
     public static final String PROPERTY_CLUSTER = "CLUSTER";
     public static final String PROPERTY_MESSAGE_TYPE = "MSG_TYPE";
+    public static final String PROPERTY_SPECIFY_DELAY_TIME = "SPECIFY_DELAY_TIME";
```

Comment:
Maybe "CUSTOM_DELAY_TIME" is a better name?

Reply:
ok